worker/main.ml

```diff
@@ -40,10 +39,21 @@ let connect_addr =
 
 let capacity =
   Arg.value
-  @@ Arg.opt Arg.int 20
+  @@ Arg.opt Arg.int 15
   @@ Arg.info ~doc:"The number of builds that can run in parallel" ~docv:"N"
        [ "capacity" ]
 
+let internal_workers =
+  Arg.value
+  @@ Arg.opt Arg.int 30
+  @@ Arg.info
+       ~doc:
+         "The number of process that can handle more solves in parallel by the \
+          solver-service. One build job (--capacity) could be splited to more \
+          process (a build job request could have more\n\
+         \             than one target platform)."
```

Comment:
> The number of sub-processes solving requests in parallel by the solver-service. One build job ($(b,--capacity)) could be distributed amongst sub-processes (a build job could have more than one target platform).

Reply:
> could be distrubuted amongst sub-processes.

The processes are spawn by demand. A process is killed when the work is done. Spawning new process is just limited by number of internal workers (--internal-workers).